Server/Models/chat_message.js

```diff
@@ -9,6 +9,11 @@ ChatMessage.init({
     primaryKey: true,
     autoIncrement: true,
   },
+  chatId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    field: 'chat_id',
+  },
```

Comment:
what does that column refer to exactly, shouldn't you create a new module to get the chat id from

Reply:
I may have missed some details due to a connection issue internet , but I'll start working on it now. Please let me know if there are any specific requirements I should consider